assets/test/e2e/test.js

```diff
@@ -0,0 +1,89 @@
+require('jasmine-beforeall');
+
+describe('', function() {
+  var commonTeardown = function() {
+    browser.executeScript('window.sessionStorage.clear();');
+    browser.executeScript('window.localStorage.clear();');
+  };
+
+  afterAll(function(){
+    // Just to be sure lets teardown at the end of EVERYTHING, and then we need to sleep to make sure it is flushed to disk
+    commonTeardown();
+    browser.driver.sleep(1000);
+  });
+
+
+  // This UI test suite expects to be run as part of hack/test-end-to-end.sh
+  // It requires the example project be created with all of its resources in order to pass
+
+  var commonSetup = function() {
+      // The default phantom window size is a mobile resolution, let's make it bigger
+      browser.driver.manage().window().setSize(1024, 768);
```

Comment:
ah shoot just remembered i tried that already and it won't work, because you have to navigate to a page before you can alter localstorage / sessionstorage